docs/source/usage/build-public.rst

```diff
@@ -16,6 +16,11 @@ build-public
     The actions are performed in this order, meaning that *internal* properties will be taken into account when matching the JSON schema.
     If a component containing nested components is deleted, those nested components are deleted as well.
 
+    .. warning::
+        **Warning!** The ``metadata.component`` will not be removed even when the JSON schema applies to it.
```

Comment:
I would remove the "Warning" as it is sufficient to have the "warning"-box:
![image](https://github.com/user-attachments/assets/71fcac89-e1c9-4ae9-8f69-7de1d9ec3f30)